### DISCUSSION TITLE
[change] Content Produced Message.

### DISCUSSION THREAD

**Discussion by VerzatileDev** [VerzatileDevOrg/Programming_HandBook#49] [Q&A] [answered]
On the website currently there is a small message on the bottom left of the page which references " Content Produced by [VerzatileDev]& [JDSherbert]  " 

![image](https://github.com/VerzatileDevOrg/Programming_HandBook/assets/47317248/405db296-610d-49d5-ba88-6ba57653c45d)

![image](https://github.com/VerzatileDevOrg/Programming_HandBook/assets/47317248/fee514e5-3062-4f17-b83c-d41d3cdd92b3)

But as we are integrating to move contributors and developers there is a thought of that it should reference as Open source and its contributors instead.

What are the Thoughts on this?
(What should the message state instead)

**Comment by VerzatileDev** [comment]
One proposed solution would be to use 
![image](https://github.com/VerzatileDevOrg/Programming_HandBook/assets/47317248/20a40aae-27ba-470e-9639-d7037cfe7f5a)

Though should it link to the Organization or the Book

**Comment by JDSherbert** [comment]
Maybe something like "Hosted at www.theURL.com"

**Reply by VerzatileDev** [reply, marked as answer]
An Open source project hosted at ..  " Would be an interesting take as an example its short, drirect to the point and avoid mentioning a collective, person, or something specifically Id say

**Comment by VerzatileDev** [comment]
Has been addressed as such ``` An Open source project hosted at: [Programming Handbook](https://github.com/VerzatileDevOrg/Programming_HandBook) ```

![image](https://github.com/VerzatileDevOrg/Programming_HandBook/assets/47317248/d62b1c44-7c6e-4aa8-88e0-a8581538d63b)

With the following closes the following discussion.
